### ISSUE TITLE
Can't format optionals

### ISSUE THREAD

**Issue by moritzstoetter** [Viatorus/emio#103]
```c++
#include <print>
#include <optional>
#include <https://viatorus.github.io/emio/dist/0.6.1/emio.hpp>

int main() {
    std::optional<int> opt = {};
    std::string str = emio::format("Optional {}.", opt);  
    std::println("{}", str);
}
```

Currently fails (https://godbolt.org/z/EE8bsnT7f), because emio can't see the specialization for `std::optional` somehow. The specialization in `include/std.hpp` is fine. Pasting it into the godbolt example, makes the error go away. So maybe it the issue lies with the order of includes?

I haven't tried any of the other specializations in `std.hpp` but that might be a starting point.

**Comment by Viatorus** [comment]
Hi Moritz, thank you for using emio. ;)

The support for std::optional (and some other std types) has been implemented in version [0.7.0](https://github.com/Viatorus/emio/releases/tag/0.7.0). You are using an older (0.6.1) version in your example.

I think this misunderstanding comes from the fact, that the ReadMe links to an older emio version in godbolt.  I updatet it.

https://github.com/Viatorus/emio/pull/104

**Comment by moritzstoetter** [comment]
Yep, that's it. Thanks for clearing that up.
